orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -185,6 +196,40 @@ def test_forward_call_positional_and_keyword_arguments():
     output = model(a, x, y, z)
     assert output is not None
 
+@pytest.mark.parametrize("forward_statement", [
+    "model(one)",
+    "model(x=one)",
+    "model(one, None, None)",
+    "model(one, None, z=None)",
+    "model(one, None)",
+    "model(x=one, y=one)",
+    "model(y=one, x=one)",
+    "model(y=one, z=None, x=one)",
+    "model(one, None, z=one)",
+    "model(x=one, z=one)",
+    "model(one, z=one)",
+    "model(one, z=one, y=one)",
+    "model(one, one, one)",
+    "model(one, None, one)",
+    "model(z=one, x=one, y=one)",
+    "model(z=one, x=one, y=None)"
+])
+def test_compare_pytorch_forward_call_positional_and_keyword_arguments(forward_statement):
+    one = torch.FloatTensor([1])
+
+    model = NeuralNetSimplePositionalAndKeywordArguments()
+    pytorch_result = eval(forward_statement + ".item()")
+
+    model = NeuralNetSimplePositionalAndKeywordArguments()
+    model = ORTModule(model)
+    ortmodule_result = eval(forward_statement)
+    # TODO: remove backward call when the issue with multiple call to forward fixed.
+    ortmodule_result.backward()
+    ortmodule_result = ortmodule_result.item()
+    ortmodule_result_again = eval(forward_statement + ".item()")
+    assert ortmodule_result == ortmodule_result_again
```

Comment:
shouldnt it be numpy's `assert_allclose` due to the float nature of the output?

Reply:
It should be exactly the same computation. And all values have exact representation is floating point.
If this exact equality fail one day, it means something is broken in the code.